corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -77,6 +82,21 @@ def _get_case_ids(self):
 
 
 class MigrationTestCase(BaseMigrationTestCase):
+    def test_migration_blacklist(self):
+        COUCH_SQL_MIGRATION_BLACKLIST.set(self.domain_name, True, NAMESPACE_DOMAIN)
+        with self.assertRaises(AssertionError):
+            self._do_migration(self.domain_name)
+        COUCH_SQL_MIGRATION_BLACKLIST.set(self.domain_name, False, NAMESPACE_DOMAIN)
+
+    @patch.object(HQExportSchema, "get_db", MagicMock())
```

Comment:
would it be better to patch `stale_get_export_count` and give it a nonzero return value? i'm not sure i understand how patching get_db() would create the test condition